endpoints/purchase_orders.md

```diff
@@ -262,6 +263,205 @@ curl --location --request POST 'https://api.gearbox.com.au/public/v1/purchase_or
+  group: "",               // string, optional, must match existing Vehicle Group in the system
+  service_number: ,        // integer, optional, must match existing Service number in the system
+  repair_number: ,         // integer, optional, must match existing Repair number in the system
+  tyre_number: ,           // integer, optional, must match existing Tyre number in the system
+  other_number: ,          // integer, optional, must match existing Other number in the system
+  general_ledger_code: "", // string, optional, if a general ledger code match is not found than an error is thrown
+  actual_cost: 0.0,        // float, optional, format: 100.00 - this will be calculated based on totals from purchase order items
+  actual_tax: 0.0,         // float, optional, format: 100.00 - this will be calculated based on totals from purchase order items
+  tax_rate: "",            // string, optional, must be either Inclusive, Exclusive or Not Applicable. Automatically calculated if not present.
+  purchase_order_items: [  // array, required, size must be less than or equal to 10
+    {
+        id: 1                    // integer, optional, required to update a purchase order item - will create a new purchase order item if it is not present
+        tax: 0.0,                // float, optional, format: 100.00
+        each: 0.0,               // float, optional, format: 100.00
+        site: "",                // string, optional, if a site match is not found than an error is thrown
+        total: 0.0,              // float, optional, format: 100.00
+        quantity: 0.0,           // float, optional, format: 100.00
+        part_number: "",         // string, optional, if a match is not found a new Part will be created
+        tax_percent: 0.0,        // float, optional, format: 100.00
+        invoice_date: "",        // date, optional, format: 'yyyy-mm-dd'
+        invoice_number: "",      // string, optional, maximum length of 20 characters
+        part_description: "",    // string, optional
+        received_quantity: 0,    // float, optional, format: 100.00
+        general_ledger_code: "", // string, optional, if a general ledger code match is not found than an error is thrown
+        tax_rate: ""             // string, optional, must be either Inclusive, Exclusive or Not Applicable. Automatically calculated if not present.
+    }
+  ]
+}
+```
+
+### Response status codes:
+
+- 200: OK
+- 400: Bad Request
+- 401: Unauthorised
+- 403: Forbidden
+- 404: Not Found
+- 413: Request Entity Too Large
+- 415: Unsupported media type
+- 422: Unprocessable Entity
+
+### 200 - Successful response
+
+```JSON
+{
+  "id": 123,
+  "purchase_order_item_ids": [123, 124]
+}
+```
+
+###### Example
+
+<details>
+<summary>Updating purchase order and creating new purchase order item</summary>
+
+```
+curl --location --request PATCH 'https://api.gearbox.com.au/public/v1/purchase_orders/1' \
```

Comment:
I think the curl request examples on this and the `update purchase order item` one are switched. We include a `purchase_order_item.id` when updating a `purchase_order_item`, but we don't when we're creating a new purchase order item.